### PR TITLE
New Author: Luis Beltran

### PR DESCRIPTION
**Pull request by icebeam7**
:pencil2: **Blog Url**: https://luisbeltran.mx

:scroll: **Feed Url**: https://luisbeltran.mx/feed/

By sending this pull request to add my blog I verify that I adhere to the Planet Xamarin blog guidelines (please check each item that applies):

- [X] I have a valid blog & RSS URL, both using HTTPS :lock: with a valid certificate
- [X] I host NO malicious or offensive content on the blog (including photos, swearing, etc.)
- [X] My blog is active with at least 3 Xamarin related blog posts in the last 6 months
- [X] In addition to the previous guideline; it should be apperant that the blog is active for a longer period of time and the posts on it are not fabricated in a small amount of time to comply to the '3 posts in last 6 months' guideline. _NOTE: I had [a previous blog](https://icebeamwp.blogspot.mx) where I shared technical content, but decided to move to wordpress from this month onwards. I am commited to publish a Xamarin post on a weekly/biweekly basis. The other blog will not be active anymore._
- [ ] I have applied a filter (Optional: if blog is only about Xamarin or you use `xamarin` in the title or categories in your blog posts)
- [X] I have added my Author class to the csproj file
- [X] I have implemented `IAmACommunityMember` or one of the other derivatives
- [X] I understand, if I delete my blog, I will be deleted from Planet Xamarin
- [X] I understand, my blog may be removed at any time if any of these guidelines are broken.
- [X] The submitted blog is not owned by a company which sole purpose it is to advertise a product of productline.

Only the collaborators of this repository decide whether a blog is eligible or not. They will determine if the rules are followed and if the quality of the content is high enough. 

Opening another PR, after the first one is closed in a small period of time will not be reevaluated. If a PR is closed, the issues with your blog are too significant to be fixed within a matter of hours.

### CODE REVIEW

**Comment by icebeam7** [comment]
Hi. Could somebody please check this pull request? :)

**Comment by jfversluis** [comment]
Was the other blog on Planet Xamarin? Did you remove it?

Also I'm not completely sure on the filter. In theory it would be possible to post something about UWP which is not about Xamarin and it will still end up on the aggregator?

File: src/Firehose.Web/Authors/LuisBeltran.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      public string GravatarHash => "1e1d718f3c761611f1ff2540ad9d4c22";
+      public string GitHubHandle => "icebeam7";
+      public GeoPosition Position => new GeoPosition(20.487380, -100.952348);
+      public Uri WebSite => new Uri("https://luisbeltran.mx/");
+      public IEnumerable<Uri> FeedUris { get { yield return new Uri("https://luisbeltran.mx/feed/"); } }
+      public string FeedLanguageCode => "es";
+    
+      public bool Filter(SyndicationItem item)
+      {
+          var allowedCategories = new[] { "xamarin", "android", "ios" };
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/LuisBeltran.cs]: Are you planning on writing about android and iOS without mentioning xamarin in the categories?

If not, just remove the entire filtering. We already filter for "xamarin".

File: src/Firehose.Web/Authors/LuisBeltran.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+using System.ServiceModel.Syndication;
+using System.Web;
+
+namespace Firehose.Web.Authors
+{
+  public class LuisBeltran : IAmAMicrosoftMVP, IFilterMyBlogPosts
+  {
+      public string FirstName => "Luis";
+      public string LastName => "Beltran";
+      public string ShortBioOrTagLine => "is a passionate Mexican software community guy who enjoys learning, talking and helping others about Xamarin, Azure, and C#";
```
Comment (Cheesebaron) [inline src/Firehose.Web/Authors/LuisBeltran.cs]: "is a passionate Mexican software community guy, who enjoys teaching, talking and helping others about Xamarin, Azure, and C#."

Is more correct English.

**Comment by icebeam7** [comment]
Thanks for your recommendation @Cheesebaron . The description has been updated and the filter removed.

**Comment by Cheesebaron** [comment]
Welcome to PlanetXamarin. You can go ahead and add linkback to our web-page on your blog now.

**Comment by icebeam7** [comment]
Thank you! Sure thing, I'm going to do that :)

**Comment by icebeam7** [comment]
Hello. Very quick question. I published a new Xamarin post on my blog two days ago but it was not detected/mentioned by the Twitter account. Do I have to configure anything else? Thanks!

**Pull request merged**